analysed-packages/libpng/version-1.6.39/libpng-1.6.39-SPDX2TV.spdx

```diff
@@ -0,0 +1,11016 @@
+SPDXID: SPDXRef-item494710
+FileChecksum: SHA1: 5987241f8c795fd68f3da65d58b32d602c687b1f
+FileChecksum: SHA256: 2bb03c06c367d3250a692c1e93ceb12a7ee9e6473176646f5bf4ed0a7cb41562
+FileChecksum: MD5: 8ad59547447effb31a209e58be3b5190
+LicenseConcluded: (LicenseRef-BSD-3-Clause OR LicenseRef-GPL-2.0-or-later)
+LicenseComments: <text> (The information in the file is:
+
+This software is provided "as is," without warranty of any kind,
+express or implied. In no event shall the author or contributors
+be held liable for any damages arising in any way from the use of
+this software.
+
+The contents of this file are DUAL-LICENSED. You may modify and/or
+redistribute this software according to the terms of one of the
+following two licenses (at your option):
+
+
+LICENSE 1 ("BSD-like with advertising clause"):
+
+Permission is granted to anyone to use this software for any purpose,
+including commercial applications, and to alter it and redistribute
+it freely, subject to the following restrictions:
+
+1. Redistributions of source code must retain the above copyright
+notice, disclaimer, and this list of conditions.
+2. Redistributions in binary form must reproduce the above copyright
+notice, disclaimer, and this list of conditions in the documenta-
+tion and/or other materials provided with the distribution.
+3. All advertising materials mentioning features or use of this
+software must display the following acknowledgment:
+
+This product includes software developed by Greg Roelofs
+and contributors for the book, "PNG: The Definitive Guide,"
+published by O'Reilly and Associates.
+
+
+LICENSE 2 (GNU GPL v2 or later):
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software Foundation,
+Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+
+This is a dual license case. We concluded BSD-3-Clause OR GPL-2.0-or-later) </text>
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-BSD-3-Clause
+LicenseInfoInFile: LicenseRef-Dual-license
+LicenseInfoInFile: LicenseRef-BSD-style
+LicenseInfoInFile: LicenseRef-GPL-2.0
+LicenseInfoInFile: LicenseRef-gpl-2.0-plus
+LicenseInfoInFile: LicenseRef-unknown-license-reference
+LicenseInfoInFile: LicenseRef-greg-roelofs
+FileCopyrightText: <text> Copyright (c) 1998-2008, 2017 Greg Roelofs. All rights reserved. </text>
+FileAttributionText: <text> To the extend files may be licensed under BSD-3-Clause or GPL-2.0-or-later, 
```

Comment:
can you please add the following statement as FileAttribution text (I already added this in the Disclosure document, 
Please update also the yam, json and rdf files accordingly
"
BSD-3-Clause

This product includes software developed by Greg Roelofs
and contributors for the book, "PNG: The Definitive Guide,"
published by O'Reilly and Associates.
"

Reply:
I have updated the acknowledgement for BSD-3-Clause in fosslogy for all those files and regenerated the files.It is like below
https://github.com/Open-Source-Compliance/package-analysis/blob/2aca4c086212c5e250aa0c6cd1152a2c0ea0e6c9/analysed-packages/libpng/version-1.6.39/libpng-1.6.39-SPDX2TV.spdx#L820

 or should it include exactly BSD-3-Clause on top of this acknowledgement?

Reply:
@ParianGol I checked it, it is fine. You do not need to include the BSD-3-Clause on top to the acknowledgment. The license text is part of the concluded License.